src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj

```diff
@@ -17,8 +17,11 @@
     <PackageReference Include="Azure.Data.AppConfiguration" Version="1.3.0-beta.2" />
     <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.3.0" />
```

Comment:
we're releasing this with a major version update right?

Reply:
IF you agree on it, I will sync it with Zhenlan and Drago

Reply:
How about we take this offline. @RichardChen820, it would be great if you could put together a proposal or list options, any breaking changes, etc.